frame/contracts/Cargo.toml

```diff
@@ -27,8 +27,9 @@ smallvec = { version = "1", default-features = false, features = [
 wasmi-validation = { version = "0.4", default-features = false }
 
 # Only used in benchmarking to generate random contract code
-rand = { version = "0.8", optional = true, default-features = false }
-rand_pcg = { version = "0.3", optional = true }
+libsecp256k1 = { version = "0.3.5", optional = true, default-features = false, features = ["hmac"] }
+rand = { version = "0.7.3", optional = true, default-features = false }
+rand_pcg = { version = "0.2", optional = true }
```

Comment:
In the benchmarks I need to create a SecretKey. `libsecp256k1` of version"0.3.5" requires some struct that implements the `RNG` trait. In a new version of `rand`, this trait has been renamed. So I downgraded the version of `rand` because it is used only in benchmarks.

Reply:
I see `libsecp256k1` depends on an older version of `rand`. Fair enough. We should really bump the version there or re-export the crate.